src/main/resources/static/css/navbar.css

```
@@ -10,6 +10,10 @@
     border: 2px solid purple;
 }
 
+.navbar button{
+
+}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)